addons/payment_sips/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright Eezee-It
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Worldline SIPS Payment Acquiring  for online payments',
+    'version': '1.0',
+    'author': 'Eezee-It',
+    'category': 'Hidden',
+    'description': """
+Worldline SIPS Payment Acquirer for online payments
+
+Works with Worldline keys version 2.0, contains implementation of
+payments acquirer using Worldline SIPS.""",
+    'depends': ['payment'],
+    'data': [
+        'views/sips.xml',
+        'views/payment_acquirer.xml',
+        'data/sips.xml',
+    ],
+    'demo': [
+    ],
+    'active': False,
```

Comment:
This is based in payments_paypal, so, just like payment_paypal category was set Hidden